pkg/chunk/aws/dynamodb_storage_client.go

```diff
@@ -749,6 +753,10 @@ func (b dynamoDBWriteBatch) Add(tableName, hashValue string, rangeValue []byte,
 	})
 }
 
+func (b dynamoDBWriteBatch) Delete(tableName, hashValue string, rangeValue []byte) {
+	panic("implement me")
```

Comment:
If a backend isn't implemented yet be sure to check for it when the purger starts in your other PR.